t2.lic

```diff
@@ -122,9 +123,9 @@ before_dying do
     stop_script(script_name) if Script.running?(script_name)
   end
   # Don't bother releasing cyclic for NMU guilds
-  fput('release cyclic') unless %w[Commoner Barbarian Thief].include?(DRStats.guild)
+  DRCA.release_cyclics unless %w[Commoner Barbarian Thief].include?(DRStats.guild)
```

Comment:
If we're going to use functionality in `DRCA`, then it needs to be added to the `custom_require` call on line 8.